src/NzbDrone.Core/Extras/ExtraService.cs

```diff
@@ -120,6 +129,68 @@ private void ImportExtraFiles(LocalEpisode localEpisode, EpisodeFile episodeFile
                     _logger.Warn(ex, "Failed to import extra file: {0}", matchingFilename);
                 }
             }
+
+            if (!wantSubs || hasSub)
```

Comment:
Why wouldn't we want to search deeper if we already found a subtitle file?

I think this is bringing too much subtitles specific logic into `ExtraService`, instead we should lift this logic up and get all same level extra files as well as all files in a subfolder then let the specific managers decide if they want to handle the file.

If we only want to handle it for subtitles we can decide that where we're specifically dealing with subtitles instead of only handling subtitles logic here, but still asking each manager to handle the files.